core/storage/exploration/gae_models_test.py

```diff
@@ -150,11 +150,16 @@ class ExplorationRightsModelUnitTest(test_utils.GenericTestBase):
     EXPLORATION_ID_2 = '2'
     EXPLORATION_ID_3 = '3'
     EXPLORATION_ID_4 = '4'
-    USER_ID_1 = 'id_1'  # Related to all three explorations
-    USER_ID_2 = 'id_2'  # Related to a subset of the three explorations
-    USER_ID_3 = 'id_3'  # Related to no explorations
-    USER_ID_4 = 'id_4'  # Related to one collection and then removed from it
-    USER_ID_COMMITTER = 'id_5'  # User id used in commits
+    # Related to all three explorations.
```

Comment:
This and similar change are done because I introduced a new lint check to prevent trailing/inline comments unless the reason is a valid allowed pragma.